include/chemfiles/formats/MMTF.hpp

```diff
@@ -52,9 +55,17 @@ class MMTFFormat final: public Format {
     /// Current atom being read. Ranges from [0, structure.numAtoms)
     size_t atomIndex_ = 0;
 
-    /// Number of atoms read before the current model being read.
-    /// Used as an offset for adding bonds when reading models.
+    /// Number of atoms before the current model.
     size_t atomSkip_ = 0;
+
+    // Since MMTF uses model->chain->residue->atom as storage model, and
+    // chemfiles do not enforce that residues contains contiguous atoms, the
+    // atoms can be re-ordered when adding them to a MMTF structure. This vector
+    // stores the correspondance chemfiles index => MMTF index to be able to add
+    // the right bonds.
+    //
+    // This is only used when writing
```

Comment:
Ok, good. I've a few performance improvements that I should be posting soon and I need to reading to work as before.